src/date-picker/calendar/__tests__/calendar.test.tsx

```diff
@@ -211,11 +205,6 @@ describe('Date picker calendar', () => {
         ['2012-11', 'November 2012'],
       ])("for input '%s', '%s' is expected", checkMonthAndYear);
     });
-
-    it.each(['', '2012-', '2012-03', '2012-03-0', '2012-03-1'])(
```

Comment:
The partial date can be now partially parsed. That does not change the behaviour of the date picker but now when the calendar component is used as standalone, the displaying date property is now uncontrolled so the value is parsed instead.